libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -405,21 +405,20 @@ void SoaringController::update_active_state()
             case ActiveStatus::MANUAL_MODE_CHANGE:
                 // It's enabled, but wasn't on the last loop.
                 gcs().send_text(MAV_SEVERITY_INFO, "Soaring: Enabled, manual mode changes.");
-                set_throttle_suppressed(true);
```

Comment:
Why are these no longer required?

Reply:
The same line is run inside init_cruising(), called down below.